File: app/src/main/java/com/neome/api/meta/base/dto/StudioMapOfLayoutFormEditorComposite.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import java.util.Map;

import com.neome.api.meta.base.Types.MetaIdLayoutFormEditorComposite;
import com.neome.api.meta.base.dto.StudioBase;
import com.neome.api.meta.base.dto.StudioDtoLayoutFormEditorComposite;

@SuppressWarnings("unused")
public class StudioMapOfLayoutFormEditorComposite extends StudioBase
{
  public MetaIdLayoutFormEditorComposite[] keys;

  public Map<MetaIdLayoutFormEditorComposite, StudioDtoLayoutFormEditorComposite> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioMapOfLayoutFormEditorComposite.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdLayoutFormEditorComposite
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioDtoLayoutFormEditorComposite

interface StudioMapOfLayoutFormEditorComposite : StudioBase
{
  val keys: List<MetaIdLayoutFormEditorComposite>
  val map: Map<MetaIdLayoutFormEditorComposite, StudioDtoLayoutFormEditorComposite>
}